### DISCUSSION TITLE
Mark as decision

### DISCUSSION THREAD

**Discussion by kevincannon** [community/community#4697] [Discussions]
I'm really excited to use Discussions to get more out of Slack, and into a better async channels.

One feature we'd really love is a nice way to mark something as a decision. Maybe when that happens, the specific post is pinned to the top somehow so people can easily find & see it.

Or maybe there's already a good workflow for this I don't know about?

**Comment by laymonage** [comment]
Have you tried using a Q&A category?

**Reply by kevincannon** [reply]
Does that work like that? I'll check it out.
